src/lv_widgets/lv_keyboard.c

```diff
@@ -87,17 +127,23 @@ static const lv_btnmatrix_ctrl_t default_kb_ctrl_num_map[] = {
 };
 /* clang-format on */
 
-static const char * * kb_map[4] = {
+static const char * * kb_map[] = {
     (const char * *)default_kb_map_lc,
     (const char * *)default_kb_map_uc,
     (const char * *)default_kb_map_spec,
     (const char * *)default_kb_map_num
+    #if LV_USE_ARABIC_PERSIAN_CHARS == 1
+    ,(const char * *)default_kb_map_ar
```

Comment:
The comma should be placed after `num`, not on the following line.

Reply:
why?
the last element in any group isn't followed by a comma 
and the num is the last element
if we didn't use ARABIC_CHARS

Reply:
The compiler doesn't mind an extra comma at the end of the last element, and it looks neater.